dandiapi/api/models/asset.py

```diff
@@ -34,14 +34,11 @@ class AssetBlob(TimeStampedModel):
         blank=True, storage=_get_asset_blob_storage, upload_to=_get_asset_blob_prefix
     )
     sha256 = models.CharField(max_length=64, validators=[RegexValidator(f'^{SHA256_REGEX}$')])
+    size = models.BigIntegerField()
```

Comment:
```suggestion
    size = models.PositiveBigIntegerField()
```

This does require Django 3.1: https://docs.djangoproject.com/en/3.1/ref/models/fields/#positivebigintegerfield

Reply:
Sadly this was added to `django-stubs` in November, but the last release was in October. There will be some editor confusion until that makes it in.